docs/changelog/100938.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 100938
+summary: "Set includeShardsStats = true in NodesStatsRequest where the caller does not use shards-level statistics"
```

Comment:
Just to double check, do you mean here `includeShardsStats = false`?

Reply:
Yes, it's my mistake. Thanks for your check. It's fixed in f72732ea279284ddc62712184ae139fa3a3c6e60 and this PR title is also fixed.